.evergreen.yml

```diff
@@ -67,7 +67,15 @@ functions:
       params:
         working_dir: spruce/evergreen
         command: make configure-mongod
-
+    - command: shell.exec
+      type: setup
+      params:
+        shell: bash
+        script: |
+          ${PREPARE_SHELL}
+          cd $PROJECT_DIRECTORY
+          curl -LO ${mongodb_tools_url};
+          tar -zxvf mongodb-database-tools-ubuntu2204-x86_64-100.8.0.tgz
```

Comment:
There's a decompress expansion that can be used here. 
https://github.com/evergreen-ci/spruce/blob/fac923813b334096562e29b1d4de9f6d4a58d1d2/.evergreen.yml#L56